deployments/mainnet/weth/migrations/1718698838_add_wbtc_as_collateral.ts

```diff
@@ -0,0 +1,157 @@
+    const _wbtcScalingPriceFeed = await deploymentManager.deploy(
+      'WBTC:priceFeed',
+      'pricefeeds/WBTCPriceFeed.sol',
+      [
+        WBTC_BTC_PRICE_FEED_ADDRESS,  // WBTC / BTC price feed
+        BTC_ETH_PRICE_FEED_ADDRESS,   // BTC / ETH price feed 
+        8,                            // decimals
+      ]
+    );
+    return { wbtcScalingPriceFeed: _wbtcScalingPriceFeed.address };
+  },
+
+  async enact(deploymentManager: DeploymentManager, _, { wbtcScalingPriceFeed }) {
+    const trace = deploymentManager.tracer();
+
+    const WBTC = await deploymentManager.existing(
+      'WBTC',
+      WBTC_ADDRESS,
+      'mainnet',
+      'contracts/ERC20.sol:ERC20'
+    );
+
+    const wbtcPricefeed = await deploymentManager.existing(
+      'WBTC:priceFeed',
+      wbtcScalingPriceFeed,
+      'mainnet'
+    );
+
+    const {
+      governor,
+      comet,
+      cometAdmin,
+      configurator,
+    } = await deploymentManager.getContracts();
+
+    const wbtcAssetConfig = {
+      asset: WBTC.address,
+      priceFeed: wbtcPricefeed.address,
+      decimals: await WBTC.decimals(),
+      borrowCollateralFactor: exp(0.80, 18),
+      liquidateCollateralFactor: exp(0.85, 18),
+      liquidationFactor: exp(0.95, 18),
+      supplyCap: exp(1_000, 8), 
+    };
+
+    const mainnetActions = [
+      // 1. Add ezETH as asset
+      {
+        contract: configurator,
+        signature: 'addAsset(address,(address,address,uint8,uint64,uint64,uint64,uint128))',
+        args: [comet.address, wbtcAssetConfig],
+      },
+      // 2. Deploy and upgrade to a new version of Comet
+      {
+        contract: cometAdmin,
+        signature: 'deployAndUpgradeTo(address,address)',
+        args: [configurator.address, comet.address],
+      },
+    ];
+
+    const description = '# Add WBTC as collateral into cWETHv3 on Mainnet\n\n## Proposal summary\n\nCompound Growth Program [AlphaGrowth] proposes to add WBTC into cWETHv3 on Ethereum network. This proposal takes the governance steps recommended and necessary to update a Compound III WETH market on Ethereum. Simulations have confirmed the market’s readiness, as much as possible, using the [Comet scenario suite](https://github.com/compound-finance/comet/tree/main/scenario). The new parameters include setting the risk parameters based off of the [recommendations from Gauntlet](https://www.comp.xyz/t/add-wbtc-to-weth-comets-on-ethereum-and-arbitrum/5332/1).\n\nFurther detailed information can be found on the corresponding [proposal pull request](https://github.com/compound-finance/comet/pull/868), [deploy market GitHub action run](https://github.com/woof-software/comet/actions/runs/9562411576/job/26358732848) and [forum discussion](https://www.comp.xyz/t/add-wbtc-to-weth-comets-on-ethereum-and-arbitrum/5332).\n\n\n## Proposal Actions\n\nThe first proposal action adds WBTC asset as collateral with the corresponding configuration.\n\nThe second action deploys and upgrades Comet to a new version.';
```

Comment:
Remove `, [deploy market GitHub action run](https://github.com/woof-software/comet/actions/runs/9562411576/job/26358732848)` as there is no deployment